app/frontend/src/components/Markdown.tsx

```diff
@@ -1,26 +1,68 @@
-import { Box, BoxProps, makeStyles } from "@material-ui/core";
-import MarkdownIt from "markdown-it";
-import React from "react";
+import "@toast-ui/editor/dist/toastui-editor-viewer.css";
 
-const md = new MarkdownIt().disable("image");
+import { makeStyles } from "@material-ui/core";
+import ToastUIViewer from "@toast-ui/editor/dist/toastui-editor-viewer";
+import { useEffect, useRef } from "react";
 
-interface MarkdownProps extends BoxProps {
+interface MarkdownProps {
   source: string;
+  topHeaderLevel?: number;
 }
 
 const useStyles = makeStyles((theme) => ({
   root: {
```

Comment:
This currently displays things with special formatting like bold, italic etc in a different font, so I think you need to nest these styles in `.tui-editor-contents` like what you did for the editor.